impala/sasl_compat.py

```diff
@@ -0,0 +1,72 @@
+try:
+    import sasl
+
+    def build_sasl_factory(host, use_ldap, username, password, service):
```

Comment:
Factory for sasl_factory is probably overkill, but it allows to have all sasl-related code in one file.